r/vignettes/developers/install_details.Rmd

```diff
@@ -57,23 +57,31 @@ correct components are described below.
 
 The diagram below shows how the R package finds a libarrow installation on Windows.
 
-```{r, echo=FALSE, out.width="70%"}
+```{r, echo=FALSE, out.width="70%", fig.alt = "Flowchart of libarrow installation on Windows systems - find full description on page"}
```

Comment:
Would you mind adding the full subheader title to this alt text? That way it would be a little bit easier to skim with a screen reader and then get the heading one wants?